dev/io.openliberty.checkpoint_fat_mail/publish/servers/imapTestServer/server.xml

```diff
@@ -0,0 +1,28 @@
+<server>
+  <featureManager>
+    <feature>componentTest-1.0</feature>
+    <feature>servlet-4.0</feature>
+    <feature>javaMail-1.6</feature>
+  </featureManager>
+
+  <mailSession>
+    <mailSessionID>testIMAPMailSession</mailSessionID>
+    <jndiName>IMAPTestingApp/IMAPMailSessionServlet/testIMAPMailSession</jndiName>
+    <description>mailSession for testing IMAP protocol</description>
+    <storeProtocol>imap</storeProtocol>
+    <host>localhost</host>
+    <user>imap@testserver.com</user>
+    <password>${imap_password}</password>
+    <from>smtp@testserver.com</from>
+    <property name="mail.imap.host" value="localhost" />
```

Comment:
Sorry I didn't think about this before.  What if you make this a variable:
```xml
<property name="mail.imap.host" value="$imap_host" />
```
and set `IMAP_HOST=localhost` in the `server.env` on restore?

Reply:
done!